plenum/server/consensus/ordering_service.py

```diff
@@ -2310,6 +2307,9 @@ def process_old_view_preprepare_reply(self, msg: OldViewPrePrepareReply, sender)
         if result != PROCESS:
             return result, reason
 
+        if sender != self._data.primary_name:
```

Comment:
Why do we need this check? The OldViewPrePrepares are not created by `self._data.primary_name`, since they have been created by a Primary from the previous view.
We have https://jira.hyperledger.org/browse/INDY-2239 for correct dealing with the issue.